### ISSUE TITLE
Target size minimum clarification(s)

### ISSUE THREAD

**Issue by scottaohara** [w3c/wcag#3727]
Many code editors allow users to expand and collapse blocks of code.  These toggles are often portrayed as arrows/chevrons, and their sizing / spacing in relation to other code block toggles is generally contingent on the font-size / line-height of the code editor.

While there is a lot of overlap to the 'inline' exception here, it seems like maybe this scenario is out of scope for that exception since the toggles aren't really "inline".  More like inline-adjacent.

I'd imagine this scenario might better fit under the "essential", since the sizing/spacing of these toggles can't be made any bigger without impacting the line-height of the code editor.  I think that makes sense.  Arguably a user could increase the spacing of the toggles by adjusting the line-height and/or font size of the code editor... and while that is a bit of a round-about way of doing that, it did make me wonder if users were provided settings to adjust target size, if that would be a sufficient way to pass this SC.

So, to summarize:
1. is my understanding that the toggles would go under the 'essential' exception correct, or would this scenario make sense to clarify/callout under the 'inline' exception?
2. Does providing users a setting or settings to adjust target sizing/spacing count as means to pass this SC?  And if not, it might be good to explain why.

**Comment by mraccess77** [comment]
I created an issue as well to understand what is meant by line height and to figure out if it’s specific to that css property or more broadly used.

**Comment by gundulaniemann** [comment]
Just my two cent: The requirements as well as the exceptions come from the end user's experience point of view. Consequently, the inline exception should be applied only if the height is determined by text height by nature. Nothing hinders to define collapsed blocks at an adequate height to also ensure a sufficient target size or at least spacing. Tis is also needed to tell the blocks apart.

**Comment by mraccess77** [comment]
Personally, I think it's still unclear how a person knows that something is controlled by the the line height of non-target text.  Is some easy way to check for this? Could someone write out instructions on how a tester would check for it?

**Comment by bruce-usab** [comment]
On backlog call, we concur that scotts 1 is covered by inline exception.  Also that 2. is not relevant.  @scottaohara volunteered to draft something.

**Comment by scottaohara** [comment]
in the PR i'm going to make for this, it was also discussed that i'd touch on https://github.com/w3c/wcag/issues/4540 - and particularly what is considered `text`

**Comment by bruce-usab** [comment]
@scottaohara for your second question:
> Does providing users a setting or settings to adjust target sizing/spacing count as means to pass this SC?

That reads to me as a straightforward way to meet one of the other exceptions for [SC 2.5.8](https://www.w3.org/TR/WCAG22/#target-size-minimum):
> **Equivalent:**  The function can be achieved through a different control on the same page that meets this criterion;

That exception was written to allow for an alternative to the undersized target, but it literally tracks.  The setting adjustment would need to be on the same page.

The approach would also be permissible as a conforming alternative versions, but pointing to an exception in the criterion seems cleaner.

**Comment by patrickhlauke** [comment]
if the control to change the sizing/spacing was not directly on the same page (but, say, in a user settings section of a site, where it can then be enabled/disabled for the whole site), then that would possibly fall more under conforming alternative version.

i think the main caveat is that it needs to be a setting that can be turned on/off, not something that gets automatically done based on things outside of the user's direct control (for instance, some naive "if this is a 'touch' device - because it has a small viewport or whatever other indirect inference - then make targets bigger"), as otherwise it won't be something that a user can explicitly enable when they need it

**Comment by patrickhlauke** [comment]
> That exception was written to allow for an alternative to the undersized target, but it literally tracks. 

not quite literally ... as it's not going to be a different control that allows for the function to be achieved...but rather a different control that then makes the *original* control bigger ... unless you squint and pretend that this replaces the original control with the bigger control, which is not what *actually* happens

**Comment by bruce-usab** [comment]
> I created an issue as well to understand what is meant by line height and to figure out if it’s specific to that css property or more broadly used.

@mraccess77 please hit me with a clue bat, as I am not finding it.

**Comment by patrickhlauke** [comment]
@bruce-usab I believe @mraccess77 is referring to https://github.com/w3c/wcag/issues/3633 which has since been moved to a discussion https://github.com/w3c/wcag/discussions/3829

**Comment by mraccess77** [comment]
@bruce-usab https://github.com/w3c/wcag/discussions/3829
